### DISCUSSION TITLE
copilot.vim

### DISCUSSION THREAD

**Discussion by molleweide** [NTBBloodbath/doom-nvim#219] [General]
Tim pope has create `copilot.vim`. just wanted to mention

**Comment by NTBBloodbath** [comment]
Hi, I don't personally like copilot and I would like to avoid it in every possible way lol.

This is the reason of why I'm not going to include this in Doom, however you're still able to use it in your custom plugins setup. Copilot plugin shouldn't die because of Doom so I guess it's fine to setup it by yourself :eyes:

**Reply by molleweide** [reply]
Cool it is interesting to hear ur thoughts. I havent used myself. Ive heard very varying things about it. Some very good, som bad.

**Reply by molleweide** [reply]
Hey would you mind sharing what you dont like with copilot?
